shared/chat/create-channel/index.desktop.js

```diff
@@ -53,21 +54,20 @@ const CreateChannel = (props: Props) => (
 
 const _inputStyle = {
   ...globalStyles.flexBoxRow,
-  marginTop: globalMargins.large,
+  marginTop: globalMargins.medium,
 }
 
 const _buttonsStyle = {
   ...globalStyles.flexBoxRow,
-  marginTop: globalMargins.large,
+  marginTop: globalMargins.xlarge,
 }
 
 const _boxStyle = {
   ...globalStyles.flexBoxColumn,
   alignItems: 'center',
-  height: '100%',
```

Comment:
i think we need this for scrolling to work correctly

Reply:
Hmm - scroll is still working. And width and position no longer seem necessary either. I changed the size of the modal window to be a fixed size, centered instead of a stretching 100% box. So I think we no longer need this.